index.js

```diff
@@ -1,15 +1,21 @@
 export const attention = {
   base: 'border-2 relative',
   tooltip:
-    'bg-gray-700 border-gray-700 text-white rounded-4 py-6 px-8',
-  callout: 'bg-green-100 border-green-400 py-8 px-16 rounded-8',
+    'i-bg-$color-tooltip-background i-shadow-$shadow-tooltip-shadow i-text-$color-tooltip-text rounded-4 py-6 px-8',
+  callout: 'i-bg-$color-callout-background i-border-$color-callout-border i-text-$color-callout-text py-8 px-16 rounded-8',
   popover:
-    'bg-white border-white rounded-8 p-16 filter drop-shadow-20',
+    'i-bg-$color-popover-background i-text-$color-popover-paragraph-text i-shadow-$shadow-popover rounded-8 p-16 filter',
```

Comment:
`filter` is deprecated and won't have any effect here. It relied on another class, which also is deprecated: `drop-shadow-20`. They made the arrow in the popover get a shadow, too:
<img width="202" alt="Screenshot 2023-05-22 at 14 34 02" src="https://github.com/warp-ds/component-classes/assets/41303231/0894c939-50da-4a39-8570-84238f9b3f5c">


If we replace those with a box-shadow property (as `i-shadow-$...` class applies), the result will be more like this I believe:
<img width="184" alt="Screenshot 2023-05-22 at 14 35 46" src="https://github.com/warp-ds/component-classes/assets/41303231/e5cf0c08-8f7c-4e7b-a78d-af7db5669ad8">

Is this correct?

Reply:
yes exactly! I'm looking at that now